### PR TITLE
PIL-1054: Deploy label modification

### PR DESCRIPTION
**Pull request by divyaBacancy**

### CODE REVIEW

File: src/screens/Home/components/ChainListItem.js [hunk 1]
```diff
@@ -39,23 +38,19 @@ export type Props = {|
...
-        <DeployContainer onPress={onPressDeploy}>
-          <DeployValue>{t('button.deploy')}</DeployValue>
-          <Icon name="question" width={14} height={14} color={colors.labelTertiary} />
+      {!isDeployed ? (
+        <DeployContainer>
+          <TitleWithIcon onPress={onPress}>{title}</TitleWithIcon>
+          <HazardIcon name="warning" width={16} height={16} color={colors.hazardIconColor} onPress={onPressDeploy} />
         </DeployContainer>
+      ) : (
+        <Title>{title}</Title>
```
Comment (poocart) [inline src/screens/Home/components/ChainListItem.js:51]: IMO using ternary for conditional rendering is hard to read when it grows. Can we render here with basic logical operators?

**Review by poocart — APPROVED** [review]
LGTM, just small comment.

File: src/screens/Home/components/ChainListItem.js [hunk 2]
```diff
@@ -68,23 +63,28 @@ const Container = styled.TouchableOpacity`
...
+`;
+
 const Title = styled(Text)`
   flex: 1;
   margin: ${spacing.mediumLarge}px 0 ${spacing.mediumLarge}px 36px;
   ${fontStyles.medium};
 `;
 
-const Value = styled(Text)`
+const TitleWithIcon = styled(Text)`
```
Comment (mdjastrzebski) [inline src/screens/Home/components/ChainListItem.js:82]: The name is a bit confusing, as there is no icon in this particular component.

File: src/screens/Home/components/ChainListItem.js [hunk 3]
```diff
@@ -39,23 +38,19 @@ export type Props = {|
...
-      )}
-
-      {!isDeployed && (
-        <DeployContainer onPress={onPressDeploy}>
-          <DeployValue>{t('button.deploy')}</DeployValue>
-          <Icon name="question" width={14} height={14} color={colors.labelTertiary} />
+      {!isDeployed ? (
+        <DeployContainer>
+          <TitleWithIcon onPress={onPress}>{title}</TitleWithIcon>
+          <HazardIcon name="warning" width={16} height={16} color={colors.hazardIconColor} onPress={onPressDeploy} />
```
Comment (mdjastrzebski) [inline src/screens/Home/components/ChainListItem.js:48]: As the icon is small you probably need to add thing like `hitSlop={hitSlop20}` for user to be able to trigger the action.

File: src/screens/Home/components/ChainListItem.js (hunk 2 above)
Comment (mdjastrzebski) [reply src/screens/Home/components/ChainListItem.js:82]: Seem like if you removed `flex: 1` from Title you could share the same style with that.

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]
Generally looks good, some minor tweaks suggested.

**Review by mdjastrzebski — APPROVED** [review]
I intended to select approve

File: src/screens/Home/components/ChainListItem.js (hunk 1 above)
Comment (divyaBacancy) [reply src/screens/Home/components/ChainListItem.js]: Done.

File: src/screens/Home/components/ChainListItem.js (hunk 2 above)
Comment (divyaBacancy) [reply src/screens/Home/components/ChainListItem.js]: Done.

File: src/screens/Home/components/ChainListItem.js (hunk 3 above)
Comment (divyaBacancy) [reply src/screens/Home/components/ChainListItem.js]: Done.

**Pull request merged**